File: src/main/java/club/xiaojiawei/hearthstone/entity/ExtraEntity.java
package club.xiaojiawei.hearthstone.entity;

import lombok.Data;
import lombok.ToString;

/**
 * @author 肖嘉威
 * @date 2022/11/29 11:46
 */
@Data
@ToString(callSuper = true)
public class ExtraEntity extends CommonEntity {

    private ExtraCard extraCard = new ExtraCard();
}

File: Hearthstone-Script/src/main/java/club/xiaojiawei/bean/log/ExtraEntity.kt
package club.xiaojiawei.bean.log;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

/**
 * @author 肖嘉威
 * @date 2022/11/29 11:46
 */
@EqualsAndHashCode(callSuper = true)
@Data
@ToString(callSuper = true)
public class ExtraEntity extends CommonEntity {

    public ExtraCard extraCard = new ExtraCard();

}
